compiler/ballerina-parser/src/main/java/io/ballerinalang/compiler/internal/parser/BallerinaParser.java

```diff
@@ -5209,6 +5235,88 @@ private boolean isUnaryOperator(SyntaxKind kind) {
         }
     }
 
+    /**
+     * Parse array type descriptor.
+     * <p>
+     * <code>
+     * array-type-descriptor := member-type-descriptor [ [ array-length ] ]
+     * member-type-descriptor := type-descriptor
+     * array-length :=
+     *    int-literal
+     *    | constant-reference-expr
+     *    | inferred-array-length
+     * inferred-array-length := *
+     * </code>
+     * </p>
+     * @param typeDescriptorNode
```

Comment:
keep a newline between the description and the params/returns in doc comments.

Reply:
fixed